how-to/security/install-openvpn.md

```diff
@@ -100,13 +100,28 @@ Since the client certificates and keys are only required on the client machine,
 Included with your OpenVPN installation are these (and many more) sample configuration files:
 
 ```bash
-root@server:/# ls -l /usr/share/doc/openvpn/examples/sample-config-files/
-total 68
--rw-r--r-- 1 root root 3427 2011-07-04 15:09 client.conf
--rw-r--r-- 1 root root 4141 2011-07-04 15:09 server.conf.gz
+root@server:/# ls /usr/share/doc/openvpn/examples/sample-config-files/*
+/usr/share/doc/openvpn/examples/sample-config-files/client.conf
+/usr/share/doc/openvpn/examples/sample-config-files/server.conf
 ```
 
-Start by copying and unpacking `server.conf.gz` to `/etc/openvpn/server.conf`:
+If these files under `/usr/share/doc/*` are not available:
+
+> Official minimal environments like Ubuntu Docker are optimized to never install stuff like documentation files (you probably also have not `sudo` there, etc).
+> To re-install the necessary documentation, at this point you can run these commands, as the root user:
+>
+> ```bash
+> echo 'path-include=/usr/share/doc/openvpn/examples/*' > /etc/dpkg/dpkg.cfg.d/my-openvpn
+> apt install --reinstall openvpn
+> ```
+
+Start by copying the example server configuration to `/etc/openvpn/server.conf`:
+
+```bash
+sudo cp /usr/share/doc/openvpn/examples/sample-config-files/server.conf /etc/openvpn/myserver.conf
+```
+
+In Ubuntu 20.04 or older, do this instead:
```

Comment:
We don't need to worry about 20.04. If it's different in 22.04, then it's worth adding a note. But if both 22.04 and 24.04 have the same file and can be used with the same set of instructions, then even better (and I think that's the case, the file is uncompressed in 22.04 and 24.04, right?).

Reply:
Yes.